ios/BT/bridge/ENExposureModule.m

```diff
@@ -0,0 +1,19 @@
+#import <React/RCTLog.h>
+#import <React/RCTBridgeModule.h>
+#import "BT-Swift.h"
+
+@interface ENExposureModule: NSObject <RCTBridgeModule>
+@end
+
+@implementation ENExposureModule
+
+RCT_EXPORT_MODULE();
+
+RCT_REMAP_METHOD(fetchLastDetectionDate,
```

Comment:
Do we need to introduce a new module for this? does this want to live in an existing module?

Reply:
Maybe exposure history is a fit? What do you think @mattThousand Will move there in the meantime